README.md

```diff
@@ -29,9 +29,9 @@ In order to start the dashboard locally, you will need to setup a few processes
 
  _For a detailed guide on how to achieve this, see more instructions [here](https://github.com/aragon/court-subgraph)._
 
-You can also setup the [court services](https://github.com/aragonone/court-backend/tree/master/packages/services), which include some automation for auto-revealing commitments, penalty settlements and heartbeats. In order for auto-reveals to work, you will need also to setup the [court server](https://github.com/aragonone/court-backend/tree/master/packages/server).
+You can also setup the [protocol services](https://github.com/aragonone/court-backend/tree/master/packages/services), which include some automation for auto-revealing commitments, penalty settlements and heartbeats. In order for auto-reveals to work, you will need also to setup the [protocol server](https://github.com/aragonone/court-backend/tree/master/packages/server).
 
-To start populating the court with disputes, you can use the [CLI tool](https://github.com/aragonone/court-backend/tree/master/packages/cli).
+To start populating the protocol with disputes, you can use the [CLI tool](https://github.com/aragonone/court-backend/tree/master/packages/cli).
```

Comment:
```suggestion
To start populating the protocol with disputes, you can use the [CLI tool](https://github.com/aragon/protocol-backend/tree/master/packages/cli).
```

Reply:
ah dind' change the links because just in case the repo dind't exist, did you guys already moved to that repo?

Reply:
Yes, it's already up!